bsp/esp32_s3_eye/include/bsp/config.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+/**************************************************************************************************
+ * BSP configuration
+ **************************************************************************************************/
+// By default, this BSP is shipped with LVGL graphical library. Enabling this option will exclude it.
+// If you want to use BSP without LVGL, select BSP version with '-noglib' suffix
+#if !defined(BSP_CONFIG_NO_GRAPHIC_LIB) // Check if the symbol is not coming from compiler definitions (-D...)
+#define BSP_CONFIG_NO_GRAPHIC_LIB (0)
```

Comment:
Can we add into KConfig?

Reply:
There are two usage options:
1. The user selects `noglib` BSP version: Enabling graphic library does not make sense, the required `esp_lvgl_port` is not included in this verision of the BSP
2. The user selects standard BSP version: Selecting `BSP_CONFIG_NO_GRAPHICAL_LIB` does not make too much sense. The `esp_lvgl_port` is already present, only the BSP graphical functions are not available

So IMO, having this option hidden is better. Do you see a usecase where the user might want to change this?

Reply:
You are right, we don't need it.